plugins/plugin-maven/che-plugin-maven-server/src/main/java/org/eclipse/che/plugin/maven/server/core/project/PomChangeListener.java

```diff
@@ -71,6 +72,16 @@ public void onEvent(ProjectItemModifiedEvent event) {
                 }
             }
         });
+
+        eventService.subscribe(new EventSubscriber<PomModifiedEventDto>() {
```

Comment:
FYI: since EventService is synchronous long running subscriber can break file events analysis and reaction for further subscribers.